components/runtime/squeak5/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
```

Comment:
The same copyright issue here

Reply:
I think you refer to a copyright issue of the Makefile.    (in the oi-userland framework).   There is also the license issue of the squeak work itself.   I have found an additional license file which I downloaded from http://www.squeak.org if you click on "LICENSE" it also has some info.   I'm going to include that as well ... However this point may be 'moot' as I think your remark on 'copyright issue' is essentially about the oi-userland Makefile.